README.md

```diff
@@ -23,13 +23,37 @@ practices.
 ## Versioning
 The commit hash of the deployed code is considered its version identifier. The commit hash can be retrieved locally via `git rev-parse HEAD`.
 
-## Setting Up
-
-Contile uses Rust, and requires the latest stable iteration. See
-[rustup.rs](https://rustup.rs/) for how to install this application.
-
-Once Rust is installed you can compile using `cargo build`. This will
-create a development release.
+## Setting Up Your Local Dev Environment
+1. Install Rust. See [rustup.rs](https://rustup.rs/) for how to install on your platform.
+2. Compile Contile using `cargo build`. This will create a development release.
+3. Create connection to ADM server:
+    1. Contile requires a connection to the ADM servers. The contract
+    tests include a stub application that can provide ADM-like returns. See the 
+    [adm_settings_test.json](./adm_settings_test.json) file for reference.
+    2. Run the following from the root of the Contile repository to start the ADM stub:
+    ```shell
+        docker run \
+        --env PORT=5000 \
+        --env RESPONSES_DIR=/tmp/partner/ \
+        --env ACCEPTED_MOBILE_FORM_FACTORS=phone,tablet \
+        --env ACCEPTED_DESKTOP_FORM_FACTORS=desktop \
+        -v `pwd`/test-engineering/contract-tests/volumes/partner:/tmp/partner \
+        -p 5000:5000 \
+        mozilla/contile-integration-tests-partner
+    ```
+4. Configuration settings are contained in the sample `sa-test.toml` file at the root of the Contile repository.  
+You may change settings [there](sa-test.toml) that pertain to your local development on Contile.
+5. Start the local dev application by running: 
+```shell
+ #! /bin/bash
+RUST_LOG=contile=trace,config=debug \
+    cargo run -- --config sa-test.toml #--debug-settings
+```
```

Comment:
Step 4 is kind of unnecessary. Someone reading the command can probably extrapolate that `sa-test.toml` contains test configurations.
```suggestion
4. Start the local dev application by running: 
```shell
 #! /bin/bash
RUST_LOG=contile=trace,config=debug \
    cargo run -- --config sa-test.toml #--debug-settings
```
```

Reply:
If it's ok with you, I'd kind of want to this one to stay. It's just a small flag in the `cargo run` command, which someone might not immediately see and I'd personally appreciate this context as a dev if I were getting set up. You ok if it remains? It read really nicely in the runbook

Reply:
Hmm, the extra step makes it look like you need to take some sort of action on step 4. The action that the user needs to take is unclear (i.e. is there something that they update something in the file? how do they know they've accomplished the step properly?). So, if we want to keep the extra context, let's merge the steps like:

```
  4. Start the local dev application with the configurations specified in `sa-test.toml`.
  ```shell
   #! /bin/bash
  RUST_LOG=contile=trace,config=debug \
      cargo run -- --config sa-test.toml 
  ``
  You may change settings that pertain to your local development on Contile directly in the file.
```